Automattic/woocommerce-payments
# Clean up old settings code related to the grouped settings flag.

Fixes #2050 

#### Changes proposed in this Pull Request

This PR removes references to the `_wcpay_feature_grouped_settings` option as well as `WC_Payments_Features::is_grouped_settings_enabled()`. Refactors some code where necessary, and updated/removed tests that related to grouped settings. It also removes some front end code referencing custom and branded buttons that utilize old settings options.

I think I've removed everything I needed to and nothing that I didn't but would appreciate a thorough review on this.

#### Testing instructions

Check to make sure that all "Grouped Settings" features are working properly.

-------------------


**Post merge**
